src/proof/alethe/alethe_proof_rule.h

```diff
@@ -445,6 +445,8 @@ enum class AletheRule : uint32_t
   BV_BITBLAST_STEP_BVEQUAL,
   BV_BITBLAST_STEP_CONCAT,
   BV_BITBLAST_STEP_CONST,
+
+  BV_BITWISE_SLICING,
```

Comment:
Ideally we wouldn't have to add a new rule and hard code Alethe here. Is this necessary?

Reply:
I think it is necessary - we need to identify exactly which proof rule was used and it's not covered by any existing rule.  Maybe I'm missing something - do you have an alternative suggestion?

Reply:
At least, I would refactor the code in the Alethe proof post processor so that the PfRule was printed as a generic identifier in the proof. I don't like the idea that every new rule we have to add in 2 enumerations (PfRule and AletheRule).

Maybe @HanielB has an idea how to do this.

Reply:
so alethe will not need any new handlers for this?

Reply:
That's right. Not at this moment.

Reply:
done